ipyparallel/client/asyncresult.py

```diff
@@ -571,6 +610,41 @@ def _display_single_result(self):
         if self.execute_result is not None:
             display(self.get())
 
+    def _display_results_only_single_result(self):
+        if get_ipython() is None:
+            # displaypub is meaningless outside IPython
+            return
+        if self.execute_result is not None:
+            display(self.get())
+
+    @check_ready
+    def display_results_only(self, groupby="type"):
```

Comment:
Maybe implement this as a `results_only=True` in `display_outputs`? That should be less duplication, I think, to just skip stream output instead of a whole new method.

Reply:
Sounds good. Will do.

Reply:
Done.